pkgs/development/libraries/gtk+/3.x.nix

```diff
@@ -21,12 +21,14 @@ stdenv.mkDerivation rec {
     sha256 = "1l45nd7ln2pnrf99vdki3l7an5wrzkbak11hnnj1w6r3fkm4xmv1";
   };
 
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
+
   nativeBuildInputs = [ pkgconfig gettext gobjectIntrospection perl ];
 
   buildInputs = [ libxkbcommon ];
   propagatedBuildInputs = with xlibs; with stdenv.lib;
-    [ expat glib cairo pango gdk_pixbuf atk at_spi2_atk ]
-    ++ optionals stdenv.isLinux [ libXrandr libXrender libXcomposite libXi libXcursor wayland ]
+    [ expat glib cairo pango gdk_pixbuf atk at_spi2_atk libXi libXfixes libXrandr libXrender libXcomposite libXcursor ]
```

Comment:
I actually just realized I can remove `libXfixes` from the list all together. It must be a propagated input of one of the other libX* libs, so when I added them to Darwin's buildInputs it because redundant. Fix is on its way.